applications/RomApplication/custom_utilities/rom_output_utility.h

```diff
@@ -0,0 +1,113 @@
+// KRATOS ┏━━━┓
```

Comment:
Implementation must be done in a cpp file.

Reply:
Sorry for the delay, I will do the corresponding modifications. Thanks.